love/src/redux/selectors/selectors.js

```diff
@@ -425,6 +427,20 @@ export const getLATISSState = (state) => {
   };
 };
 
+export const getAuthlistState = (state, subscriptions) => {
+  // const authlistData = getStreamsData(state, subscriptions);
+  // console.log(authlistData);
+  const authlistData = {
+    'event-ATDome-0-authList': { authorizedUsers: 'tribeiro@nb-tribeiro', nonAuthorizedCSCs: 'MTPtg:1' },
```

Comment:
This information must come from the backend

Reply:
I've refactored this on further commits. Thanks!